blocks/header/header.css

```diff
@@ -618,13 +619,11 @@ header nav .nav-sections > ul > li {
   header nav .nav-sections > ul > li[aria-expanded='true'] > ul.mega-menu .mega-menu-content > div,
   header nav .nav-sections > ul > li[aria-expanded='true'] > ul.mega-menu .mega-menu-content > ul {
     display: block;
-    top: calc(var(--nav-height) + 37px);
```

Comment:
Removing this seems to be breaking the leadspace margin in mobile for the home page (not sure where else yet)...

Reply:
I don't see any difference with or without this style. Since the container is `block`, `top` has no effect.